internal/handlers/pg/msg_getlog.go

```diff
@@ -85,6 +85,5 @@ func (h *Handler) MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 	if err != nil {
 		return nil, lazyerrors.Error(err)
 	}
-
```

Comment:
All other commands have an empty line there, let's keep it

Reply:
fix